tests/modules/vardictjava/main.nf

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { VARDICTJAVA } from '../../../modules/vardictjava/main.nf'
+
+workflow test_vardictjava {
+    
+    bam_input_ch = Channel.of([
```

Comment:
This need to be `Channel.value` to create a single tuple. `Channel.of`creates a channel with multiple values

Reply:
Isn't it meant to be such a channel for the input BAM files?

Reply:
Your test data is a single tuple containing meta + bam file, so essentialy a single value